customer-samples/ConsoleApp/Alpine3.10.dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
+WORKDIR /app
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy everything else and build
+COPY . ./
+RUN dotnet publish -c Release -f netcoreapp3.1 -o out
+
+# Build runtime image
+FROM mcr.microsoft.com/dotnet/core/runtime:3.1-alpine3.10
+WORKDIR /app
+COPY --from=build /app/out .
+
+# Set up Datadog APM
+RUN apk --no-cache update && apk add curl
```

Comment:
Notice here that we don't need any additional packages to run the profiler on Alpine with the new profiler, just `curl` to download the tar.